product_docs/docs/pgd/5/node_management/maintainance_with_proxies.mdx

```diff
@@ -146,7 +146,7 @@ end;
 $$ language 'plpgsql';
 ```
 
-This function takes a node name and a command and runs the command on that node, returning the results like so:
+This function takes a node name and a command and runs the command on that node, returning results like these:
```

Comment:
These suggests what follows is just results when it's the command and output.

Reply:
I'm sorry, I'm not understanding what you mean. The "like so" was okay...It tonally didn't feel like doc, so I made this edit. I don't think we need to spend too much time on this, so if you think your original was somehow clearer, it was definitely an edit I didn't feel strongly about.

Reply:
I was going for the "this is the interaction" - "these" suggests the following is all results.

Reply:
```suggestion
This function takes a node name and a command and runs the command on that node, returning the results as shown in this interaction:
```
Is that what you mean? If not, let's just change it back to "like so".